src/main/java/podz/task/Event.java

```diff
@@ -0,0 +1,50 @@
+package podz.task;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class Event extends Task {
+
+    protected String from, to, formattedFrom, formattedTo;
+
+    public Event(String description, String from, String to) {
+        super(description);
+        this.from = from;
+        this.to = to;
+        
+        String fromDT[] = from.split(" ");
+        String toDT[] = to.split(" ");
```

Comment:
Perhaps the array specifiers should be attached to the type instead of the variable.